MapboxNavigation/InstructionPresenter.swift

```diff
@@ -170,6 +231,13 @@ fileprivate struct IndexedVisualInstructionComponent {
 }
 
 extension Array where Element == VisualInstructionComponent {
```

Comment:
Should an array of `VisualInstructionComponents` be given a type name or become a class/struct? Extending `Array` like this makes me think maybe, yes.

Reply:
@akitchen You would think so, and I thought on this too, but aside from adding a `typealias VisualInstructionComponents = [VisualInstructionComponent]` (which is not great, I think "Just pluralize the noun" is kind of an anti-pattern over `[TypeName]`) I'm not sure what we could do here. Instruction Components come from properties in `VisualInstruction`, so perhaps the best place to add extra sugar would be there.

Additionally, I would like to point out that this extension ended up not getting used and thus will be deleted.

Reply:
Works for me! thanks